components/raftstore/src/store/config.rs

```diff
@@ -461,6 +460,12 @@ impl Config {
             ));
         }
 
+        if self.raft_entry_max_size.0 == 0 || self.raft_entry_max_size.0 > ReadableSize::gb(3).0 {
+            return Err(box_err!(
+                "raft entry max size should be greater than 0 and less than or equal to 3GiB"
```

Comment:
I don't think we support 3GiB. Not even 2.01GiB.

Reply:
I set this to the vlaue as `raft-max-size-per-msg`, this is just an intuitive upper bound, do you mean the hard limit should be 2GB? Sorry I don't find the related code.